publish-release.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+VERSION="v1"
+CLOUD_PROJECT="prshim"
+
+npm install
+gulp build
+
+# Log in to gcloud with service account or manually with one of the following
+# commands:
+
+# gcloud auth activate-service-account --key-file client-secret.json
+# gcloud auth login
+
+# Ensure gcloud is pointing to the correct project
+gcloud config set project $CLOUD_PROJECT
+
+# Commit projects to the repo
+gsutil cp build/* "gs://$CLOUD_PROJECT/$VERSION/"
```

Comment:
Maybe enumerate the source files explicitly to avoid inadvertently pushing extraneous stuff. You could put the list of files in an env var at the top and then use that var here.